### ISSUE TITLE
Change keybinding for Replace text (⌘H) in the Editor on Mac

### ISSUE THREAD

**Issue by ShenggaoZhu** [spyder-ide/spyder#2914]
Spyder 3.0.0b2 
Python 2.7.11 64bits, Qt 5.4.1, PyQt5 5.4.1 on Darwin

The default keyboard shortcut for Replace text is "Ctrl+H", which is "⌘H" on Mac. 
https://github.com/spyder-ide/spyder/blob/master/spyderlib/config/main.py#L548

However,  "⌘H" is already bound to the menu "python -> Hide Spyder". In fact, "⌘H" on Mac is the system default shortcut to "Hide the windows of the front app".

Thus I would suggest changing the shortcut for Replace text on Mac to, for example, "Command + Alt + F" (and maybe also changing it to "Control + Alt + F" on other platforms).

**Comment by ccordoba12** [comment]
What if we change that shortcut to `Cmd+R` on Mac and `Ctrl+R` for other OSes. Is there a problem with that?

**Comment by ShenggaoZhu** [comment]
``Cmd+R`` has no conflict with other default shortcuts in Spyder. It could be a good choice as it's easy to remember and 33% shorter than ``Cmd+Alt+F``!

**Comment by ccordoba12** [comment]
Yep, that's nice and also used in other text editors (like Kate). I'll change it for beta3.

**Comment by ShenggaoZhu** [comment]
Thanks!  :+1:

**Comment by Nodd** [comment]
Ctrl+f is common in lots of applications on windows and linux. The default could be changed for mac only ?

**Comment by ccordoba12** [comment]
@Nodd, `Ctrl+F` is for searching and `Ctrl+R` for search and replace :-)

**Comment by Nodd** [comment]
Ctrl+h for search and replace is widely used, sorry.

**Comment by ShenggaoZhu** [comment]
@Nodd Here is the list of the default shortcuts for "Search and replace":
https://en.wikipedia.org/wiki/Table_of_keyboard_shortcuts#Text_editing
Windows: Ctrl+H
Mac: Cmd+F
GNOME: Ctrl+H
KDE: Ctrl+R

So ``Ctrl+H`` is the very popular on windows and gnome. However, on Mac ``Alt+Cmd+F`` is very common in many text editors, such as Xcode, TextEdit and Sublime Text.

Thus I guess we may have two options for Spyder:

1. Use the same shortcut on all platforms, and ``Ctrl+R`` (``Cmd+R`` on Mac) could be a good candidate since ``Ctrl+H`` is excluded for Mac.

2. Continue using ``Ctrl+H`` on windows and linux, but use a different one for Mac. In that case, I would suggest ``Alt+Cmd+F`` for Mac convention.

**Comment by ptoche** [comment]
sorry for the newbie question, but when will this feature be available in a ``dmg``? The current dmg was updated 22 days ago, according to the download site, while this merge is from 5 days ago, should I expect to wait a few days or should I build from source? Thanks!

**Comment by Nodd** [comment]
@ptoche You should be able to change the shortcut in the preferences, no need to wait for a new version. This PR only changes the default value.

**Comment by ccordoba12** [comment]
New DMGs will be available once we release Spyder 3.0

**Comment by ptoche** [comment]
@Nodd, very true how silly of me! Thank you very much. 

After I read about the issue, I did not investigate further. The first time around I had located the "find next", "find previous", and "find text" I somehow gave up, thinking the "find and replace" would have been listed underneath. But it is listed under "replace text", fair enough. It would have taken me a microsecond to find it if I'd looked. 

Bangs head...

:-)
